app/src/components/v-collection-field-template.vue

```diff
@@ -4,20 +4,21 @@ import type { Field, Relation } from '@directus/types';
 import { toRefs } from 'vue';
 import { useI18n } from 'vue-i18n';
 
-const props = defineProps<{
-	value: string | null;
-	placeholder?: string | null;
-	disabled?: boolean;
-	collection: string | null;
-	inject: {
-		fields: Field[];
-		relations?: Relation[];
-	} | null;
-}>();
+const props = withDefaults(
+	defineProps<{
+		modelValue?: string | null;
```

Comment:
Let's use the new `defineModel` macro: https://vuejs.org/guide/components/v-model.html

Reply:
Done in https://github.com/directus/directus/pull/21855/commits/c32d3742f037fdec4e05afe00705bf21463c1abb